### DISCUSSION TITLE
Grid Edit Mode

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#219] [General]
Hi Kjell,
I am making a Villers-Bocage scenery, and I saw that in the Grid Edit Mode there are some little things to adjust.
1) "Name" field is shorter than "Short name" field. Is this due to the effective field dimension or just a little error?
2) Column dimension is fixed, You could make it adjustable, and this would resolve problem at 1).
Thank you for you wonderful work!
Federico

**Comment by kjellmf** [comment]
Thank you for the feedback. I have not prioritized the grid edit mode yet and I will start to improve it after I've released version 1. I started to work on a replacement (https://orbat-mapper.app/testgrid), but got distracted by other things.

**Reply by federicopeyrani1969** [reply]
Yes' that looks really better!

**Comment by kjellmf** [comment]
I've made some improvements recently to the grid edit mode.
